Bottom: 9b709728c62514d56aa2bc4738440ce477952f33
Top:    30b5b906884d3ff5443986555faddadf5815d2c1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 23:39:34 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ef51bc6..73c124f 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
